istioctl/pkg/ztunnelconfig/ztunnelconfig.go

```diff
@@ -543,8 +543,8 @@ func getPodNameWithNamespace(ctx cli.Context, podflag string, enforceSinglePod b
 	return podName, podNamespace, nil
 }
 
-func setupZtunnelConfigDumpWriter(kubeClient kube.CLIClient, podName, podNamespace string, out io.Writer) (*ztunnelDump.ConfigWriter, error) {
-	debug, err := extractZtunnelConfigDump(kubeClient, podName, podNamespace)
+func setupZtunnelConfigDumpWriter(kubeClient kube.CLIClient, podName, podNamespace string, out io.Writer, port int) (*ztunnelDump.ConfigWriter, error) {
```

Comment:
port validation is missing (i.e., its between 1 and 65535), but it seems to be missing for other commands as well like `istioctl proxyconfig --proxy-admin-port`.